src/java.base/share/data/currency/CurrencyData.properties

```diff
@@ -582,7 +582,7 @@ YE=YER
 # ZAMBIA
 ZM=ZMW
 # ZIMBABWE
-ZW=ZWL
+ZW=ZWL;2024-08-31-22-00-00;ZWG
```

Comment:
JDK switches the currency at the beginning of the transition period. So in this case, it's 6/25. So we don't need to even specify the transition at all, as the next build drop is beyond that date.

Reply:
Good to know. Removed the cutover dates.